### DISCUSSION TITLE
Exception in thread review_segment_maintainer error

### DISCUSSION THREAD

**Discussion by pdobrien3** [blakeblackshear/frigate#15287] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Exception in thread review_segment_maintainer:
Traceback (most recent call last):
File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
self.run()
File "/opt/frigate/frigate/review/maintainer.py", line 483, in run
024-12-01 07:28:46.004008471      self.update_existing_segment(current_segment, frame_time, [])
TypeError: update_existing_segment() missing 1 required positional argument: 'objects'

### Steps to reproduce

1. install beta version
2.
3.
...

### Version

0.15.0-beta2

### Frigate config file

```yaml
Will provide if needed but I have had a stable setup for a long time.
```

### docker-compose file or Docker CLI command

```yaml
Will provide if needed but I have had a stable setup for a long time.
```

### Relevant Frigate log output

```shell
See above
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Hikvision

### Screenshots of the Frigate UI's System metrics pages

will add this

### Any other information that may be helpful

ninw

**Comment by pdobrien3** [comment]
![IMG_0171](https://github.com/user-attachments/assets/2b924db6-cd22-4e83-8888-23f5a0835d2d)

**Comment by pdobrien3** [comment]
Top shows a zombie process every time this error occurs

**Comment by NickM-27** [comment, marked as answer]
Fixed in https://github.com/blakeblackshear/frigate/pull/15274
